backend/FwHeadless/Services/SendReceiveHelpers.cs

```diff
@@ -17,17 +19,21 @@ public record SendReceiveAuth(string Username, string Password)
         public SendReceiveAuth(FwHeadlessConfig config) : this(config.LexboxUsername, config.LexboxPassword) { }
     };
 
-    public record LfMergeBridgeResult(string Output, string ProgressMessages);
+    public record LfMergeBridgeResult(string Output, IProgress progress)
```

Comment:
I didn't think of this when I made my suggestion. But this will actually make `IProgress` a property of LfMergeBridgeResult, and if it's serialized then it will include that object, which we probably done want. I'd suggest this:
```C#
public record LfMergeBridgeResult(string Output, bool ErrorEncountered)
{
    public LfMergeBridgeResult(string output, IProgress progress): this(output, progress.ErrorEncountered);
}
```
be defining another constructor we can achieve the same result, but without adding `IProgress` as a property of the record.